### PR TITLE
add compress many encoded bytes utils

### PR DESCRIPTION
**Pull request by ChihChengLiang**
Add utils to compress many encoded Types.<tx_type> to a single compressed bytes.

### CODE REVIEW

File: contracts/libs/RollupUtils.sol [hunk 1]
```diff
@@ -578,6 +613,24 @@ library RollupUtils {
...
+        for (uint256 i = 0; i < txBytes.length; i++) {
+            Types.Transaction memory _tx = TxFromBytes(txBytes[i]);
+            _txs[i] = Tx.Transfer(
+                _tx.fromIndex,
+                _tx.toIndex,
+                _tx.amount,
+                _tx.nonce
+            );
+        }
+        return Tx.serialize(_txs);
```
Comment (ChihChengLiang) [inline contracts/libs/RollupUtils.sol:631]: To compress many Transfers, we call this function

File: test/Reddit.spec.ts [hunk 2]
```diff
@@ -387,8 +387,8 @@ contract("Reddit", async function() {
         assert.equal(errorCode, ErrorCode.NoError);
         assert.equal(newBalanceRoot, resultTo[1]);
 
-        const compressedTxs = await RollupUtilsInstance.CompressTransferFromEncoded(
-            txBytes
+        const compressedTxs = await RollupUtilsInstance.CompressManyTransferFromEncoded(
+            [txBytes]
```
Comment (ChihChengLiang) [inline test/Reddit.spec.ts:391]: An example how to use it

File: contracts/libs/RollupUtils.sol (hunk 1 above)
Comment (vaibhavchellani) [reply contracts/libs/RollupUtils.sol:631]: So this is for N txs, right?

File: test/Reddit.spec.ts (hunk 2 above)
Comment (vaibhavchellani) [reply test/Reddit.spec.ts:391]: Are `txBytes` one transaction?

Comment (ChihChengLiang) [reply test/Reddit.spec.ts:391]: Yes, it's one transaction

File: contracts/libs/RollupUtils.sol (hunk 1 above)
Comment (ChihChengLiang) [reply contracts/libs/RollupUtils.sol:631]: Yes, the output is N transactions compressed into a bytes string.

**Pull request merged**